eclipse/plugin/src/nl/tudelft/watchdog/eclipse/ui/wizards/ProjectRegistrationInputPanel.java

```diff
@@ -43,35 +78,108 @@
 		header.setFont(HEADER_FONT);
 
 		new Label(this, SWT.NONE).setText(PROJECT_DATA_REQUEST);
-		new Label(this, SWT.NONE).setText(INPUT_IS_OPTIONAL);
+		new Label(this, SWT.NONE).setText(INPUT_IS_REQUIRED);
 
-		this.inputContainer = new Composite(this, SWT.NONE);
 		GridLayout gridLayout = new GridLayout(2, true);
-		this.inputContainer.setLayout(gridLayout);
+		Composite inputFieldContainer = new Composite(this, SWT.NONE);
+		inputFieldContainer.setLayout(gridLayout);
+		this.createInputFields(inputFieldContainer);
 
-		this.createInputFields();
+		sliderComposite = new Composite(this, SWT.NONE);
+		sliderComposite.setLayout(new GridLayout(1, true));
+		createSlider(sliderComposite);
 
+		this.inputContainer = new Composite(this, SWT.NONE);
+		this.inputContainer.setLayout(gridLayout);
 		this.createButtonAndStatusContainer(CREATE_PROJECT_BUTTON_TEXT);
 	}
 
-	private void createInputFields() {
-		this.projectName = RegistrationStep.createLinkedLabelTextField(PROJECT_NAME_LABEL, PROJECT_NAME_TEXTFIELD_TOOLTIP, inputContainer);
-		this.projectWebsite = RegistrationStep.createLinkedLabelTextField(PROJECT_WEBSITE_LABEL, PROJECT_WEBSITE_TEXTFIELD_TOOLTIP, inputContainer);
+	private void createInputFields(Composite inputContainer) {
+		this.projectName = RegistrationStep.createLinkedLabelTextField(PROJECT_NAME_LABEL,
+				PROJECT_NAME_TEXTFIELD_TOOLTIP, inputContainer);
+		this.projectWebsite = RegistrationStep.createLinkedLabelTextField(PROJECT_WEBSITE_LABEL,
+				PROJECT_WEBSITE_TEXTFIELD_TOOLTIP, inputContainer);
 		this.ciUsage = RegistrationStep.createYesNoDontKnowQuestionWithLabel(CI_USAGE_LABEL_TEXT, inputContainer);
 
 		new Label(inputContainer, SWT.NONE).setText(DO_YOU_USE_STATIC_ANALYSIS);
 		// Filler to complete this grid row
 		new Composite(inputContainer, SWT.NONE);
 
-		this.codeStyleUsage = RegistrationStep.createYesNoDontKnowQuestionWithLabel(CODE_STYLE_USAGE_LABEL_TEXT, inputContainer);
-		this.bugFindingUsage = RegistrationStep.createYesNoDontKnowQuestionWithLabel(BUG_FINDING_USAGE_LABEL_TEXT, inputContainer);
-		this.automationUsage = RegistrationStep.createYesNoDontKnowQuestionWithLabel(OTHER_AUTOMATION_USAGE_LABEL_TEXT, inputContainer);
+		this.codeStyleUsage = RegistrationStep.createYesNoDontKnowQuestionWithLabel(CODE_STYLE_USAGE_LABEL_TEXT,
+				inputContainer);
+		this.bugFindingUsage = RegistrationStep.createYesNoDontKnowQuestionWithLabel(BUG_FINDING_USAGE_LABEL_TEXT,
+				inputContainer);
+		this.automationUsage = RegistrationStep.createYesNoDontKnowQuestionWithLabel(OTHER_AUTOMATION_USAGE_LABEL_TEXT,
+				inputContainer);
+
+		this.toolsUsed = RegistrationStep.createLinkedLabelTextField(TOOL_USAGE_LABEL_TEXT,
+				TOOL_USAGE_TEXTFIELD_TOOLTIP, inputContainer);
+	}
 
-		this.toolsUsed = RegistrationStep.createLinkedLabelTextField(TOOL_USAGE_LABEL_TEXT, TOOL_USAGE_TEXTFIELD_TOOLTIP, inputContainer);
+	private void createSlider(Composite composite) {
+		UIUtils.createLabel("", composite);
```

Comment:
I personally prefer to keep the style consistent in these files. Thus doing `new JLabel(composit, "")`, rather than calling an external static method.

Reply:
I agree with consistency, but am not going to tackle this for now due to the extra code required for italic/bold labels.